### PR TITLE
Polish pharmacies - remove duplicates

### PR DESCRIPTION
**Pull request by mutipg**
### id_to_remove > as_duplicate_of_id
aptekagemini-cf61fb > gemini-cf61fb
aptekarodzinna-cf61fb > rodzinna-cf61fb
aptekasloneczna-cf61fb > sloneczna-cf61fb

### CODE REVIEW

File: data/brands/amenity/pharmacy.json [hunk 1]
```diff
@@ -225,39 +225,6 @@
         "name": "Apotheka"
       }
     },
-    {
-      "displayName": "Apteka Gemini",
-      "id": "aptekagemini-cf61fb",
-      "locationSet": {"include": ["pl"]},
```
Comment (LaoshuBaby) [inline data/brands/amenity/pharmacy.json:231]: There are still a lot elements in OSM named after this according to taginfo, also "Apteka Rodzinna" and "Apteka Słoneczna"

图片

So if you only removed those 3 entry, they will be created once again, our collector will found they are still "widely used" in real world.

If this change is from business merge or rebrand, or bought by another brand, I'd suggest you add a `"matchNames": ["Apteka Gemini"],` in your targeted `gemini-cf61fb`, another 2 also need this.

For example, "頂好超市" was bought by "家樂福超市":

https://github.com/osmlab/name-suggestion-index/blob/f57288c32f5fb2de87fc6c8bc5943ba818bf403b/data/brands/shop/supermarket.json#L9696-L9701

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

**Comment by kjonosm** [comment]
@matkoniecz You seem to have a good overview of the Polish tagging preferences. Which version do you think we should keep/promote in NSI?

**Comment by matkoniecz** [comment]
here I am not really sure, so created https://community.openstreetmap.org/t/sloneczna-czy-apteka-sloneczna/6296

**Comment by Dimitar5555** [comment]
@matkoniecz Is there any result from the discussion on the forum? If there isn't, it may be beneficial to create a poll and see if there is a majority which prefers one of the options.

**Comment by matkoniecz** [comment]
Skipping "apteka" ("pharmacy") in name seems preferred but it is not unanimous.

**Comment by bhousel** [comment]
stale

**Pull request closed without merging**